srcpkgs/QMPlay2/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'QMPlay2'
+pkgname=QMPlay2
+version=18.07.03
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config qt5-qmake qt5-host-tools"
+makedepends="alsa-lib-devel ffmpeg-devel libass-devel libcdio-devel libgme-devel
+ libsidplayfp-devel libva-glx-devel libXv-devel pulseaudio-devel
+ qt5-svg-devel qt5-tools-devel qt5-x11extras-devel taglib-devel"
+short_desc="Video and audio player which can play most formats and codecs"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://zaps166.sourceforge.net/?app=QMPlay2"
+distfiles="https://github.com/zaps166/QMPlay2/archive/${version}.tar.gz"
+checksum=e9dd6caa74e0824d950c0d11b2ed71e799f0a02c375b49ff635498133ab6b45e
```

Comment:
According to https://github.com/zaps166/QMPlay2/releases , the source tarball is:
````
distfiles="https://github.com/zaps166/QMPlay2/releases/download/${version}/QMPlay2-src-${version}.tar.xz"
checksum=78cdadea21728f7a902ae240aae15481bcad2b9f34598c20ee7711feeff79122
````

Reply:
I chose the other one to avoid having to change wrksrc, but that one works fine